zerver/tests/test_migrations.py

```diff
@@ -30,391 +28,62 @@
 # As a result, we generally mark these tests as skipped once they have
 # been tested for a migration being merged.
 
+USER_ACTIVATED = 102
+USER_FULL_NAME_CHANGED = 124
 
-@skip("Fails because newer migrations have since been merged.")  # nocoverage # skipped
-class MessageEditHistoryLegacyFormats(MigrationsTestCase):
```

Comment:
What's up with these dropped tests?

Reply:
We have been keeping only one latest migration test case in `test_migrations`. GitHub crops off some lines of context above this:
```
# Important note: These tests are very expensive, and details of
# Django's database transaction model mean it does not super work to
# have a lot of migrations tested in this file at once; so we usually
# delete the old migration tests when adding a new one, so this file
# always has a single migration test in it as an example.
```

Reply:
Aha, OK.  Maybe worth a commit message note?  But that code comment is pretty explanatory, so I can see it not being needed.